src/nova_deno.ts

```diff
@@ -70,6 +77,69 @@ function watchConfigFile(workspacePath: string, filename: string) {
   compositeDisposable.add(configWatcher);
 }
 
+/**
+ * This is used to determine whether certain features, like the "Find Symbol"
+ * command, are available.
+ */
+function registerEditorWatcher() {
+  // This callback is called immediately and once. Then, it is called every time the user opens a text editor.
+  const disposable = new CompositeDisposable();
+
+  const compatibleSyntaxes = ["typescript", "tsx", "javascript", "jsx"];
+  function checkFeatureAvailability(
+    documents: readonly TextDocument[],
+  ) {
+    const syntaxes = documents.map((document) => document.syntax ?? "plain");
+    const isCompatible = compatibleSyntaxes.some((syntax) =>
+      syntaxes.includes(syntax)
+    );
+
+    if (isCompatible) {
+      // @ts-expect-error: The Nova types are outdated. This feature was added in version 5.
```

Comment:
I went looking for documentation on this `context` property and couldn't find it anywhere. But this property does exist, and logs as a `[object NovaJSConfiguration]`. It seems to be identical to the value returned from `nova.workspace.config`, which is documented. I switched this code to use `config` instead and it still seems to work great.

How did you find about this `context` property?

Reply:
Oh, I see... using the `.config` property actually modifies the config files too.

Reply:
`context` is documented here: https://docs.nova.app/extensions/commands/#custom-variables-in-when-clauses.

Its values determine whether the `Find Symbol` command is made available.

Reply:
They should document that on the workspace page. Good find!